Snowflake.Data.Tests/IntegrationTests/ConnectionMultiplePoolsIT.cs

```diff
@@ -313,24 +315,69 @@ public void TestNewConnectionPoolClean()
         [Test]
         public void TestConnectionPoolExpirationWorks()
         {
-            var connectionString = ConnectionString + "expirationTimeout=0;maxPoolSize=2";
-            var conn1 = new SnowflakeDbConnection();
-            conn1.ConnectionString = connectionString;
-            conn1.Open();
-            conn1.Close();
+            // arrange
+            const int ExpirationTimeoutInSeconds = 10;
+            var connectionString = ConnectionString + $"expirationTimeout={ExpirationTimeoutInSeconds};maxPoolSize=4;minPoolSize=2";
+            var pool = SnowflakeDbConnectionPool.GetPool(connectionString);
+            Assert.AreEqual(0, pool.GetCurrentPoolSize());
             
-            var conn2 = new SnowflakeDbConnection();
-            conn2.ConnectionString = connectionString;
-            conn2.Open();
+            // act
+            var conn1 = OpenedConnection(connectionString);
```

Comment:
Could we change the name of the method `OpenedConnection` to `OpenConnection`? `OpenedConnection` indicates, that we receive already opened connection with a specific connection string, not a new one. Same for `Async` version.

Reply:
done